src/MicrosoftTeams.ts

```diff
@@ -1566,4 +1570,73 @@ namespace microsoftTeams {
       args: args || []
     };
   }
+
+  export interface TaskInfo {
+    /**
+     * The url to be rendered in the webview/iframe.
+     */
+    url?: string;
+
+    /**
+     * JSON defining an adaptive card.
+     */
+    card?: string;
+
+    /**
+     * The requested height of the webview/iframe in pixels.
+     */
+    height?: string;
+
+    /**
+     * The requested width of the webview/iframe in pixels.
+     */
+    width?: string;
+
+    /**
+     * Title of the task module.
+     */
+    title?: string;
+
+    /**
+     * If client doesnt support the URL, the URL that needs to be opened in the browser.
+     */
+    fallbackUrl?: string;
+  }
+
+  /**
+   * Namespace to interact with the task module-specific part of the SDK.
+   * This object is usable only on the content frame.
+   */
+  export namespace tasks {
+    /**
+     * Allows an app to open the task module.
+     * @param taskInfo An object containing the parameters of the task module
+     * @param completionHandler Handler to call when the task module is completed
+     */
+    export function startTask(
+      taskInfo: TaskInfo,
+      completionHandler?: (err: string, result: string) => void
+    ): void {
+      // Ensure that the tab content is initialized
+      ensureInitialized(frameContexts.content);
+
+      let messageId = sendMessageRequest(parentWindow, "start", [taskInfo]);
+      callbacks[messageId] = completionHandler;
+    }
+
+    /**
+     * Complete the task module.
+     * @param result Contains the result to be sent to the bot or teh app. Typically a JSON object or a serialized version of it
+     * @param appId Helps to validate that the call originates from the same appId as the one that invoked the task module
```

Comment:
We are documenting for appId here but actually using appIds in the function. I am not clear on the need for appIds, but please check and correct.

Reply:
Just made the change to the app Ids yesterday. Missed changing it on the parameter. The need for appIds is if the developer has multiple apps using the same task module pages